composer.json

```diff
@@ -40,8 +40,8 @@
     },
     "require-dev": {
         "json-schema/JSON-Schema-Test-Suite": "1.1.0",
-        "phpunit/phpunit": "~3.7",
-        "phpdocumentor/phpdocumentor": "~2"
+        "phpdocumentor/phpdocumentor": "~2",
```

Comment:
@GrahamCampbell 

The change here is only related to requiring `phpunit/phpunit` with

```
$ composer require --dev --sort-packages phpunit/phpunut:^4.7.7
```